addons/checkparam/checkparam.lua

```diff
@@ -122,7 +122,13 @@ function get_text(id,data)
     local ext = extdata.decode({id=id,extdata=data})
     if ext.augments then
         for i,v in ipairs(ext.augments) do
-            split_text(id,v)
+			local stats = windower.regex.split(v,'(Pet|Avatar|Automaton|Wyvern|Luopan): ')
+			if stats[2] then
+				local pet_text = windower.regex.replace(stats[2],'\n',' ')
```

Comment:
You should probably just use `stats[2]:gsub('\n', '')`, while the regex functions are very powerful they're usually somewhat slow if you don't need them. In this case you should also be able to use `stats[2]:trim()`.

Reply:
Changed and tested with gsub in the new commit